modules/@ergonode/core/src/pages/login/index.vue

```diff
@@ -5,14 +5,24 @@
 <template>
     <Login
         @redirect-to="onRedirectTo">
-        <Component
-            :is="loginFormComponents.loginFormComponent"
-            @redirect-to="onRedirectTo" />
-        <Infographic :src="loginFormComponents.formImage" />
+        <template #form>
+            <Component
+                :is="loginFormComponents.formComponent"
+                @redirect-to="onRedirectTo" />
+        </template>
+        <FadeTransition>
```

Comment:
FadeTransition wont apply - it only works for v-if / v-show

Reply:
It's an old code , I forgot to delete it